packages/drivers/file-driver/src/fileDocumentStorageService.ts

```diff
@@ -157,44 +146,18 @@ export const FileSnapshotWriterClassFactory = <TBase extends ReaderConstructor>(
                 this.docId = versionId;
             }
 
-            if (versionId !== null && this.commitsWriter[versionId] !== undefined) {
-                // PrefetchDocumentStorageService likes to prefetch everything!
-                // Skip, as Container does not really need it.
-                throw new Error("Not supporting commit loading");
-                // Return [{id: versionId, treeId: FileStorageVersionTreeId}];
-            }
             return super.getVersions(versionId, count);
         }
 
         public async getSnapshotTree(version?: api.IVersion): Promise<api.ISnapshotTree | null> {
             if (this.latestWriterTree && (!version || version.id === "latest")) {
                 return this.latestWriterTree;
             }
-            if (version && this.commitsWriter[version.id] !== undefined) {
-                return buildSnapshotTree(this.commitsWriter[version.id].entries, this.blobsWriter);
-            }
             return super.getSnapshotTree(version);
         }
 
-        public async write(
-            tree: api.ITree,
-            parents: string[],
-            message: string,
-            ref: string): Promise<api.IVersion> {
-            let dataStoreName = ref ? ref : "container";
-            if (tree && tree.entries) {
-                tree.entries.forEach((entry) => {
-                    if (entry.path === ".component" && entry.type === api.TreeEntry.Blob) {
-                        const blob: api.IBlob = entry.value;
-                        const content = blob.contents.split(":");
-                        if (content[0] === `{"pkg"`) {
-                            dataStoreName = content[1].substring(1, content[1].lastIndexOf(`"`));
-                        }
-                    }
-                });
-            }
-
-            const commitName = `commit_${dataStoreName}`;
+        public async uploadSummaryWithContext(summary: api.ISummaryTree, context: ISummaryContext): Promise<string> {
+            const tree = convertSummaryTreeToSnapshotITree(summary);
 
             // Sort entries for easier diffing
             this.sortTree(tree);
```

Comment:
This may not be needed. Could you try removing this and see if the tests still pass?

Reply:
`convertSummaryTreeToSnapshotITree` or `uploadSummaryWithContext` in general ?

Reply:
I meant this.sortTree

Reply:
You were right, it was not needed.

Reply:
Awesome. The following may also not be needed:
delete tree.id;
removeNullTreeIds(tree);
However, this should probably be done as a follow up when we change the format of the tree in the snapshot tests from ISnapshotTree -> ISummaryTree. This will become invalid anyway and will be removed then.